package.json

```diff
@@ -22,9 +22,9 @@
     "@typescript/vfs": "1.6.0",
     "@web/dev-server-esbuild": "1.0.2",
     "@web/test-runner": "0.19.0",
-    "@zenfs/core": "0.9.7",
-    "@zenfs/dom": "0.2.6",
-    "@zenfs/zip": "0.3.1",
+    "@zenfs/core": "^1.0.0",
```

Comment:
would you mind pinning these to a patch (drop the caret)?

Reply:
Done in [`50a6a33` (#40)](https://github.com/reddit/play/pull/40/commits/50a6a3318fdd886ecc2be8dd55066d57cf2e1e37)